problemsets-mat223/linearalbegra.tex

```diff
@@ -4788,21 +4788,83 @@ \subsection*{More Change of Basis}
 	\end{parts}
 
 	\question
-	Let $\vec c_1=\mat{2\\1}$, $\vec c_2=\mat{5\\3}$, $\mathcal C=\{\vec c_1,\vec c_2\}$, and $A=\mat{2&5\\1&3}$.
+	Let $\vec c_1=\mat{2\\1}$, $\vec c_2=\mat{5\\3}$, $\mathcal C=\Set{\vec c_1,\vec c_2}$, and $A=\mat{2&5\\1&3}$.
 	Note that $A^{-1}=\mat{3&-5\\-1&2}$ and that $A$ changes vectors from the $\mathcal C$ basis to the standard
 	basis and $A^{-1}$ changes vectors from the standard basis to the $\mathcal C$ basis.
 	\begin{parts}
 		\item Compute $[\vec c_1]_{\mathcal C}$ and $[\vec c_2]_{\mathcal C}$.
+			\begin{solution}[inline]
+				$[\vec c_1]_{\mathcal C}=\mat{1\\0}$ and 
+				$[\vec c_2]_{\mathcal C}=\mat{0\\1}$.
+			\end{solution}
 	\end{parts}
 	Let $T:\R^2\to\R^2$ be the linear transformation that stretches in the $\vec c_1$ direction by a factor of $2$
 	and doesn't stretch in the $\vec c_2$ direction at all.
 	\begin{parts}[resume]
 		\item Compute $T\mat{2\\1}$ and $T\mat{5\\3}$.
+			\begin{solution}[inline]
+				$T\mat{2\\1}=\mat{4\\2}$ and $T\mat{5\\3}=\mat{5\\3}$.
```

Comment:
Add `\vec c_*` in the middle. e.g. `T\mat{2\\1} = T\vec c_1 = 2\vec c_1 = \mat{4\\2}`